### DISCUSSION TITLE
[RFC] remove specific remote cache artifact?

### DISCUSSION THREAD

**Discussion by cjayyy** [vercel/turborepo#7247] [Ideas]
Hello 👋, 

Is there any way of how to delete certain remote cache artifact? 
I know there is option to remove all cache in Vercel admin, but never saw this with more granular option. 
It would be great to have this available in CLI.

**Comment by anthonyshew** [comment]
This functionality doesn't exist in Turborepo today so I've changed this over to an RFC to see if any other folks have interest in such a feature.

**Reply by cjayyy** [reply]
Thanks. 

Do you know about any workaround? I guess there is remote cache API somewhere available, but looking into it's source code I wasn't able to find anything like this.

**Reply by anthonyshew** [reply]
You're correct; there's no "delete this specifically" mechanism in the API today.

Can I ask about your motivation for wanting such a feature?

**Reply by cjayyy** [reply]
My motivation is debugging in general. 

Although I had one specific use-case where I wanted it:
I have CI pipeline (GH actions) which builds an app using Turborepo with caching turned on. This build has release ID which is then propagated to registration service which registers the app and serves the wanted version. For some reason the registration service is unable to handle existing revision and fails if it tries to register the revision that was already registered. This is a problem when I push changes and build + register new version and revert this new version afterwards, which leads to using the first cached build with already registered release ID. 
In this case I have no other option than to push some fake changes in order to trigger new build - because I have no control over registration service and I don't have a way how to `--force` build in CI pipeline. Perhaps there is some other thing I can do which didn't come to my mind? 
Easiest thing I could think of was to delete cache for this specific build. But since I didn't want to delete all cache for other tasks/build versions I didn't know how to do that. 

Fortunately the flow is fixed now, so this is not a problem anymore for me.
